joomla/joomla-cms
# [5] Action Log: Block/Unblock user

Pull Request for Issue #41374 .

### Summary of Changes

log user block / unblock action

### Testing Instructions

block / unblock users and check actionslog

In addition, check that editing a user still works.

### Actual result BEFORE applying this Pull Request

User block / unblock action not logged.

### Expected result AFTER applying this Pull Request

User block / unblock action logged.

Editing a user still works.

### Link to documentations
Please select:
